scripts/azure-ci-test.sh

```diff
@@ -65,6 +65,11 @@ deploy_ratify() {
     --set image.repository=${ACR_NAME}.azurecr.io/test/localbuild \
     --set image.crdRepository=${ACR_NAME}.azurecr.io/test/localbuildcrd \
     --set image.tag=${TAG} \
+    --set cosign.enabled=true \
+    --set cosign.key="-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvjrMZFyaBDsvg5e0C8JaHqw8ULuc
+n947ODVAMvfdqtjqK2eW77OGrsFLdkbG3BET9U4Dj37odn4kI5lC4Lj9Eg==
+-----END PUBLIC KEY-----" \
```

Comment:
can we pass the key using the existing file similar to https://github.com/deislabs/ratify/blob/main/Makefile#L329

Reply:
yeah, I can create a .pub in the test folder,.